autoload/fzf/vim.vim

```diff
@@ -35,6 +35,14 @@ let s:bin = {
 \ 'preview': s:bin_dir.(executable('ruby') ? 'preview.rb' : 'preview.sh'),
 \ 'tags':    s:bin_dir.'tags.pl' }
 let s:TYPE = {'dict': type({}), 'funcref': type(function('call')), 'string': type(''), 'list': type([])}
+if s:is_win
+  if has('nvim')
+    let s:bin.preview = split(system('for %A in ("'.s:bin.preview.'") do echo %~sA'), "\n")[1]
```

Comment:
Can you explain how this line works? Any documentation I can refer to?

Reply:
I used a simple variant of the `FOR` command in cmd.exe and parameter extension on the for loop variable. It runs the command after `do` so [0] matches `echo %~sA` and `[1]` matches the output of that `ECHO` command. The [1] output is an absolute path with backslash for path separator so the `escape` is necessary for fzf binary to not remove the backslashes when it runs the preview command.

Aside from different path separators because of `set shellslash`, `fnamemodify` and that line should output the same absolute path.

Documentation:
https://ss64.com/nt/for2.html
https://ss64.com/nt/syntax-args.html

Reply:
There is a low risk of 8.3 filename disabled manually (by administrators) but the common case is it remains enabled to support legacy software.